app/mainwindow.cpp

```diff
@@ -1069,7 +1069,7 @@ void MainWindow::loadUrl(const QUrl &url)
         QApplication::setOverrideCursor(Qt::WaitCursor);
         this->configureStreamDecoding(in);
         m_tikzQtEditorView->editor()->setPlainText(in.readAll());
-        setCurrentEncoding(in.codec());
+        //setCurrentEncoding(in.codec());
```

Comment:
This codec stuff probably needs fixing before merging. I'm not sure what the current way of working is..